packages/coding-standard/tests/Fixer/Commenting/ParamReturnAndVarTagMalformsFixer/FixturePhp74/skip_nested_call.php.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Symplify\CodingStandard\Tests\Fixer\Commenting\ParamReturnAndVarTagMalformsFixer\FixturePhp74;
+
+class SkipNestedCall
+{
+    /**
+     * @param mixed[] $value
+     *
+     * @return mixed[]
+     */
+    public static function typeCastAll(string $type, array $value): array
+    {
+        return array_map(
+            /**
+             * @param mixed $item
+             *
+             * @return mixed
+             */
+            static fn($item) => self::typeCast($type, $item),
+            $value
+        );
+    }
+}
```

Comment:
Sorry, there was a missing method. I skipped the real implementation of it for test simplicity.

```suggestion
class SkipNestedCall
{
    /**
     * @param mixed[] $value
     *
     * @return mixed[]
     */
    public static function typeCastAll(string $type, array $value): array
    {
        return array_map(
            /**
             * @param mixed $item
             *
             * @return mixed
             */
            static fn($item) => self::typeCast($type, $item),
            $value
        );
    }

	/**
	 * @param mixed $value
	 *
	 * @return mixed
	 */
	public static function typeCast(string $type, $value)
	{
		return $value;
	}
}
```

Reply:
:thinking: ??? what's wrong with the diff?